mifosng-provider/src/main/java/org/mifosplatform/accounting/journalentry/api/JournalEntriesApiResource.java

```diff
@@ -49,7 +49,7 @@
     private static final Set<String> RESPONSE_DATA_PARAMETERS = new HashSet<>(Arrays.asList("id", "officeId", "officeName",
             "glAccountName", "glAccountId", "glAccountCode", "glAccountType", "transactionDate", "entryType", "amount", "transactionId",
             "manualEntry", "entityType", "entityId", "createdByUserId", "createdDate", "createdByUserName", "comments", "reversed",
-            "referenceNumber", "currency"));
```

Comment:
Thank you sir :)

On Mon, Apr 13, 2015 at 2:15 PM, Vishwas Babu A J <notifications@github.com>
wrote:

> In
> mifosng-provider/src/main/java/org/mifosplatform/accounting/journalentry/api/JournalEntriesApiResource.java
> <https://github.com/openMF/mifosx/pull/1347#discussion_r28222300>:
>
> > @@ -49,7 +49,7 @@
> >      private static final Set<String> RESPONSE_DATA_PARAMETERS = new HashSet<>(Arrays.asList("id", "officeId", "officeName",
> >              "glAccountName", "glAccountId", "glAccountCode", "glAccountType", "transactionDate", "entryType", "amount", "transactionId",
> >              "manualEntry", "entityType", "entityId", "createdByUserId", "createdDate", "createdByUserName", "comments", "reversed",
> > -            "referenceNumber", "currency"));
>
> good job on catching this :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openMF/mifosx/pull/1347/files#r28222300>.
>



-- 
Thank you & regards
Rajesh Reddy k
7795405419